subsys/bluetooth/controller/Kconfig

```diff
@@ -18,6 +18,13 @@ config BLECTRL_SLAVE_COUNT
 	  NOTE: the number of master roles is defined as
 	  BT_MAX_CONN - BLECTRL_SLAVE_COUNT
 
+config BLECTRL_MAX_CONN_EVENT_LEN
```

Comment:
Maybe change this to `BLECTRL_MAX_CONN_EVENT_LEN_DEFAULT`? We will later add APIs to override it

Reply:
ok